examples/cloudflare-workers/README.md

```diff
@@ -0,0 +1,28 @@
+# Using Momento inside a Cloudflare Worker
+
+This example uses
+[Wrangler](https://developers.cloudflare.com/workers/wrangler/) to create a
+typescript worker
+
+## How to use
+
+1. Clone and install the example
+
+```bash
+git clone https://github.com/momentohq/client-sdk-javascript.git
+cd client-sdk-javascript/examples/cloudflare-workers
+npm install
+```
+
+2. Create a Momento cache inside of the [console](https://console.gomomento.com)
+3. Create a [token for your cache](https://console.gomomento.com/tokens)
+4. Copy the `Auth Token`, `Rest Endpoint`, and cache name to the `wranger.toml` file
+5. Start the development server
+
+```bash
+npm run start
```

Comment:
Getting a 401 on opening the page; DM'd you about it but I just used my own token that I use through the CLI and got the endpoint by decoding the token